src/ui/rendering/handlebarsrenderer.js

```diff
@@ -194,8 +194,27 @@ export default class HandlebarsRenderer extends Renderer {
     });
 
     this.registerHelper('processTranslation', function (options) {
+      const pluralizationInfo = {};
+      const interpolationParams = {};
       let { phrase, count } = options.hash;
-      return TranslationProcessor.process(phrase, options.hash, count);
+
+      Object.entries(options.hash).forEach(([key, value]) => {
+        const isPluralFormParam = key.length > 10 && key.substring(0, 10) === 'pluralForm';
```

Comment:
I think key.startsWith is slightly cleaner here

Reply:
updated!